Sources/MongoSwift/BSON/AnyBSONValue.swift

```diff
@@ -2,8 +2,20 @@ import Foundation
 
 /// A struct wrapping a `BSONValue` type that allows for encoding/
 /// decoding `BSONValue`s of unknown type.  
-public struct AnyBSONValue: Codable, Equatable {
-    /// The `BSONValue` wrapped by this struct. 
+public struct AnyBSONValue: Codable, Equatable, Hashable {
+    // swiftlint:disable:next legacy_hashing
+    public var hashValue: Int {
```

Comment:
let's add a todo linking to the ticket for making all underlying BSON types Hashable

Reply:
Done.